web/src/pages/Home/__generated__/HomeQuery.graphql.ts

```diff
@@ -0,0 +1,83 @@
+/**
```

Comment:
files generated by relay don't need to be commited

Reply:
oh, makes sense, thanks